src/index.ts

```diff
@@ -161,6 +159,7 @@ function processExtensions(definition: DefinitionWithExtend): Definition {
             if (!ext.isModernExtend) {
                 assert.fail(`'${definition.model}' has legacy extend in modern extend`);
             }
+            // TODO: concat instead of spread push?
```

Comment:
Yes please, no clue why `push` was used here.

Reply:
Nevermind, forgot `concat` creates a new array, `push` will be faster.